@xen-orchestra/lite/src/locales/en.json

```diff
@@ -79,6 +83,7 @@
   "password": "Password",
   "password-invalid": "Password invalid",
   "pause": "Pause",
+  "please-confirm": "Please confirm.",
```

Comment:
No punctuation.

Reply:
We need it like that.

Reply:
You say it's necessary because of the Clemence model, I guess?

But I wonder if the punctuation is really intended because sometimes there is and others not.